data/json/recipes/recipe_weapon.json

```diff
@@ -2784,4 +2784,28 @@
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "leather", 30 ], [ "rope_superior_short", 2, "LIST" ] ], [ [ "scrap", 9 ], [ "razor_blade", 9 ] ] ]
   }
+  {
+    "type": "recipe",
+    "result": "makeshift_shotgun",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ [ "gun", 1 ] ],
+    "difficulty": 2,
+    "time": "2 h",
+    "autolearn": true,
+    "book_learn": [ [ "manual_shotgun", 1 ] ],
+    "qualities": [ { "id": "SAW_M", "level": 1 }, { "id": "GLARE", "level": 2 } ],
+    "tools": [
+      [
+        [ "oxy_torch", 6 ],
+        [ "welder", 30 ],
+        [ "welder_crude", 45 ],
+        [ "toolset", 45 ],
+        [ "small_repairkit", 150 ],
+        [ "large_repairkit", 30 ]
+      ]
+    ],
+    "components": [ [ [ "pipe", 2 ] ], [ [ "nail", 1 ] ], [ [ "scrap", 2 ] ] ]
```

Comment:
I'd personally add one more piece of scrap to cover the sling swivel, you can ignore this suggestion if you want.

Reply:
What's a "sling swivel"? just curious.

Reply:
![image](https://user-images.githubusercontent.com/51099123/84091695-34562880-a9c3-11ea-83e8-2f92a5686f87.png)
You feed a sling through the loop.

Reply:
Got it, thanks. It's added.